app/models/krikri/provider.rb

```diff
@@ -0,0 +1,80 @@
+module Krikri
+  ##
+  # This models provider data from the {Krikri::Repository}.
+  #
+  # @todo create an appropriate `ActiveTriples::PersistenceStrategy' for
+  #   handling providers as data from a repository alongside LDP data.
+  # @see ActiveTriples::Resource
+  class Provider < DPLA::MAP::Agent
+    configure :base_uri => 'http://dp.la/api/contributor/'
```

Comment:
Yes. And I think we need to step back and consider KriKri's expectations on data models soon.  While the Mapping DSL and the harvesters are model-agnostic, a lot of the higher-level stuff we've done in the last few weeks has hard dependencies on `DPLA::MAP::Aggregation`, this now depends on `DPLA::MAP::Agent`, and some things make assumptions about predicates used.